.github/actions/spelling/action.yml

```diff
@@ -0,0 +1,48 @@
+name: Check Spelling
```

Comment:
This is still the old name.

Also, as I asked, please document why we have that name and why it can’t be changed.

Reply:
It's not really about the name, but I have added an explanation.